bandit/blacklists/calls.py

```diff
@@ -320,6 +320,19 @@
 |      |                     | - os.tmpnam                        |           |
 +------+---------------------+------------------------------------+-----------+
 
+B704: pytorch_load_save
+
+Use of unsafe PyTorch load. `torch.load` can lead to arbitrary code execution,
+and improper use of `torch.save` might expose sensitive data or lead to data
+corruption.
+
++------+---------------------+--------------------------------------+-----------+
+| ID   |  Name               |  Calls                               |  Severity |
++======+=====================+======================================+===========+
+| B704 | pytorch_load_save|       | - torch.load                         | Medium    |
+| B704 | pytorch_load_save|       | - torch.save                         | Medium    |
++------+---------------------+--------------------------------------+-----------+
```

Comment:
This is effectively a duplicate of the plugin. The blacklist calls is meant to create simple checks that don't require any complicated logic except that a call of the give name was called. So you can remove the changes to this file.